src/modules/70-pipeline/components/execution/StepDetails/common/StepDetails/StepDetails.tsx

```diff
@@ -197,6 +198,12 @@ export function StepDetails(props: StepDetailsProps): React.ReactElement {
             <td>{timeout}</td>
           </tr>
         )}
+        {!!retryInterval && (
```

Comment:
This will not work if retryInterval is any falsy value.
```suggestion
        {!isUndefined(retryInterval) && (
```